pgbot/commands/__init__.py

```diff
@@ -94,11 +96,29 @@ async def handle(invoke_msg: discord.Message, response_msg: discord.Message = No
         return
 
     if response_msg is None:
-        response_msg = await embed_utils.send_2(
-            invoke_msg.channel,
-            title=f"Your command is being processed:",
-            fields=(("\u2800", "`Loading...`", False),),
-        )
+        confused = emotion.get("confused")
+        if confused > 300 and random.random() < confused / 6000:
```

Comment:
The 6000 on this line should prolly be 600, right?

Reply:
nah, 300 / 600 is a 50% chance of confused, that seems a bit high...
Hmm, maybe it should be higher than 5% tho, something like a 15% chance starting at 300, so.. maybe 2000?

Reply:
yea 2000 sounds about right

Reply:
okey, I'll make lé commit